<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorSwiftMonitorDialog</class>
 <widget class="QDialog" name="CSimulatorSwiftMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>swift driver monitor</string>
  </property>
  <layout class="QVBoxLayout" name="vl_Settings">
   <item>
    <widget class="QTabWidget" name="tw_SwiftMonitorDialog">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_AircraftSituation">
      <attribute name="title">
       <string>Situation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="BlackGui::Editors::CSituationForm" name="editor_Situation">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Parts">
      <attribute name="title">
       <string>Parts</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Parts">
       <item>
        <widget class="QGroupBox" name="gb_Parts">
         <property name="title">
          <string>Parts</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxParts">
          <item>
           <widget class="BlackGui::Editors::CAircraftPartsForm" name="editor_AircraftParts">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>125</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Simulator">
      <attribute name="title">
       <string>Simulator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gb_General">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="fl_General">
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_SimulatorPlugin">
            <property name="text">
             <string>Simulator plugin:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_SimulatorPlugin">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_Simulator">
            <property name="text">
             <string>Simulator:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="le_Simulator">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_Connected">
            <property name="text">
             <string>Connected:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_Simulating">
            <property name="text">
             <string>Simulating:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbl_Paused">
            <property name="text">
             <string>Paused:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cb_Connected">
            <property name="text">
             <string>simulator connected?</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="cb_Simulating">
            <property name="text">
             <string>simulator simulating?</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="cb_Paused">
            <property name="text">
             <string>simulator paused?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_LogMessages">
      <attribute name="title">
       <string>Log information</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_LogInformation">
       <item>
        <widget class="QGroupBox" name="gb_LogMessages">
         <property name="title">
          <string>Simulator API calls</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxLogMessages">
          <item>
           <widget class="BlackGui::Components::CLogComponent" name="comp_LogComponent"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="gb_Settings">
         <property name="title">
          <string>Simulator settings</string>
         </property>
         <layout class="QVBoxLayout" name="vl_GroupBoxSettings">
          <item>
           <widget class="BlackGui::Components::CSettingsSwiftPlugin" name="comp_Settings"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="vs_Situation">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_SwiftMonitorDialog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSettingsSwiftPlugin</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsswiftplugin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CLogComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/logcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CSituationForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/situationform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CAircraftPartsForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/aircraftpartsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bb_SwiftMonitorDialog</sender>
   <signal>accepted()</signal>
   <receiver>CSimulatorSwiftMonitorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_SwiftMonitorDialog</sender>
   <signal>rejected()</signal>
   <receiver>CSimulatorSwiftMonitorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
